README.md

```diff
@@ -1 +1,7 @@
-# Hello-Microverse2
\ No newline at end of file
+Hello microverse is the first project in week one
+i made this with html
+![Screenshot from 2021-06-08 12-46-58](https://user-images.githubusercontent.com/75973193/121163891-d53a2680-c857-11eb-9747-f21344944e00.png)
+From the project i learned;
+To use git & github flow
+To setup linters
+To write descritive readme files and pull request descriptions
```

Comment:
- [ ] I believe you can still improve your Readme file. You've added the project description and screenshot, which is nice. I think you should also add the `Built with` and the `Getting Started` sections. These are the important sections that a readme should have. [This template](https://github.com/microverseinc/readme-template) might help.

- [ ] I believe you can do better with your commit messages. Messages like `Hello`, `Linter`, `NewCommit` etc are not descriptive enough, even you might find it hard to understand what change/feature you've made such commit for. Kindly use descriptive commit messages. [This article](https://chris.beams.io/posts/git-commit/) might help. Please take note of this against your subsequent commits.
- [ ] Your branch name (`micro`) is also not descriptive. Although you can change the branch name, doing so will close this PR, which is not a good practice. However, you should ensure that meaningful and descriptive branch names that reflect the feature implemented in such branches are used in subsequent projects (e.g. in this case 'hello_microverse_feature' or 'hello_microverse', etc).
- [ ] The `.gitignore` file is completely missing in your project. You need to create  a .gitignore file that lists the `test.md` file, and `node_modules` directory (as it can be added by accident while setting up linters). Please check the project requirement in case of any doubt.